### PR TITLE
xfail currently failing projects.

### PR DESCRIPTION
**Pull request by varungandhi-apple**
See https://ci.swift.org/job/swift-main-source-compat-suite-debug//4049/consoleText

### CODE REVIEW

File: projects.json [hunk 1]
```diff
@@ -2076,6 +2118,13 @@
         "destination": "generic/platform=tvOS",
         "configuration": "Release",
         "tags": "sourcekit-disabled"
```
Comment (shahmishal) [inline projects.json:2120]: missing `,`

File: projects.json [hunk 2]
```diff
@@ -2726,6 +2775,14 @@
         "scheme": "Surge-iOS",
         "destination": "generic/platform=iOS",
         "configuration": "Release"
```
Comment (shahmishal) [inline projects.json:2777]: here too

File: projects.json [hunk 3]
```diff
@@ -2741,13 +2798,29 @@
         "scheme": "Surge-tvOS",
         "destination": "generic/platform=tvOS",
         "configuration": "Release"
+        "xfail": [
```
Comment (shahmishal) [inline projects.json:2801]: here too

File: projects.json [hunk 4]
```diff
@@ -2741,13 +2798,29 @@
...
+           }
+        ]
       },
       {
         "action": "BuildXcodeProjectScheme",
         "project": "Surge.xcodeproj",
         "scheme": "Surge-watchOS",
         "destination": "generic/platform=watchOS",
         "configuration": "Release"
+        "xfail": [
```
Comment (shahmishal) [inline projects.json:2816]: here too

**Review by shahmishal — COMMENTED** [review]
Missing few `,` in the json file.

**Comment by varungandhi-apple** [comment]
Fixed missing commas.

**Comment by varungandhi-apple** [comment]
Replaced stray tab with spaces.

**Comment by varungandhi-apple** [comment]
@swift-ci test

**Comment by varungandhi-apple** [comment]
The failure is due to the same arm64e issue.

```
error: compile command failed due to signal 6 (use -v to see invocation)
<unknown>:0: remark: did not find a prebuilt standard library for target 'arm64e-apple-macos' compatible with this Swift compiler; building it may take a few minutes, but it should only happen once for this combination of compiler and target
SIL verification failed: external declarations of SILFunctions with shared visibility is not allowed: SingleFunction || !hasSharedVisibility(RefF->getLinkage()) || RefF->hasForeignBody()
```

**Comment by varungandhi-apple** [comment]
@swift-ci test

**Comment by shahmishal** [comment]
@varungandhi-apple What is the current status of this PR?

**Comment by varungandhi-apple** [comment]
Closing this for now, other projects have been turned off in the meantime.

**Pull request closed without merging**